cost-analyzer/charts/prometheus/values.yaml

```diff
@@ -1216,6 +1216,62 @@ serverFiles:
             regex: (.+)
             action: replace
 
+      # A scrape configuration for running Prometheus on a Kubernetes cluster.
+      # This uses separate scrape configs for cluster components (i.e. API server, node)
+      # and services to allow each to use different authentication configs.
+      #
+      # Kubernetes labels will be added as Prometheus labels on metrics via the
+      # `labelmap` relabeling action.
+
+      - job_name: 'kubernetes-nodes'
```

Comment:
I would have expected this PR to simply add a new metric to the whitelist, but looks like we need to add an entirely new scrape endpoint. Can you share more?

Reply:
I tend to agree. @nickcurie can you check if it's on the cadvisor metrics we already scrape? You can learn about that job here: https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/charts/prometheus/values.yaml#L1165

Reply:
This metric isn't a cadvisor metric, its a kubelet metric. See [docs](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#metrics-in-kubernetes): "In most cases metrics are available on `/metrics` endpoint of the HTTP server ... Note that kubelet also exposes metrics in `/metrics/cadvisor`, `/metrics/resource` and `/metrics/probes` endpoints. Those metrics do not have same lifecycle."

I'm not sure if we already scrape the standard kubelet metrics or not, just wanted to point this out so we're on the same page. The query I ran in https://github.com/kubecost/cost-analyzer-helm-chart/pull/1713#discussion_r984888548 shows which metrics endpoint this comes from.

Reply:
Unfortunately, `/metrics/cadvisor` doesn't have any metrics on PVs or PVCs. `/metrics` does, so that's why we need to scrape this endpoint. I'm not sure we have another way to monitor PV usage.

Reply:
SG, just wanted to make sure we checked.